resources/js/components/Crud/CrudInput.vue

```diff
@@ -9,6 +9,7 @@ defineProps<{
   label?: string
```

Comment:
I don't use it in #title on the UsersPanel and AdminsPanel pages because it automatically takes up 100% of the space, which is problematic when I want to place two inputs (first name, last name) side by side.

Reply:
I fixed QuizHeader and added a format prop to CrudInput.